doc/needs_restarting.rst

```diff
@@ -38,10 +38,6 @@ Note that in most cases a process should survive update of its binary and librar
 .. note::
    Needs-restarting will try to guess the boot time using three different methods:
```

Comment:
s/three/two

Reply:
Good catch. I fixed it and force-pushed.